daml/Marketplace/Rule/AllocationAccount.daml

```diff
@@ -20,7 +20,6 @@ template AllocationAccountRule
   where
     signatory operator, provider, account.owner
     observer insert nominee observers
-    ensure nominee /= account.owner
```

Comment:
Whats the motivation for removing this check ?

Reply:
this was getting in the way of parties offering themselves certain services - for example a Custodian becoming an Issuer via their own role contract requires an allocation account with the Custodian as a nominee, and this allows them to use an account that they own